docs/resources/sfs_turbo.md

```diff
@@ -57,6 +57,9 @@ The following arguments are supported:
 * `crypt_key_id` - (Optional, String, ForceNew) Specifies the ID of a KMS key to encrypt the file system.
   Changing this will create a new resource.
 
+* `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id of the file system.
+  Changing this will create a new resource.
+
```

Comment:
the `docs/data-sources/enterprise_project.md` should be updated as while.

Reply:
fixed